packages/block-editor/src/components/block-heading-level-dropdown/stories/index.story.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelDropdown from '../';
+
+const meta = {
+	title: 'BlockEditor/HeadingLevelDropdown',
+	component: HeadingLevelDropdown,
+	argTypes: {
+		value: {
+			control: { type: null },
+			description: 'The currently selected heading level.',
+		},
+		options: {
+			control: 'array',
+			description:
+				'An array of supported heading levels, e.g., [1, 2, 3, 4, 5, 6].',
+		},
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
+			description:
+				'Callback triggered when a new heading level is selected.',
+		},
+	},
+	decorators: [
+		( Story ) => (
+			<div style={ { padding: '20px', maxWidth: '300px' } }>
+				<Story />
+			</div>
+		),
+	],
+};
+
+export default meta;
+
+export const Default = {
+	render: function Template( { onChange, ...args } ) {
+		const [ value, setValue ] = useState( args.value );
+
+		return (
+			<HeadingLevelDropdown
+				{ ...args }
+				value={ value }
+				onChange={ ( ...changeArgs ) => {
+					setValue( ...changeArgs );
+					onChange( ...changeArgs );
+				} }
+			/>
+		);
+	},
+	args: {
+		value: 2,
+		options: [ 1, 2, 3, 4, 5, 6 ],
```

Comment:
```suggestion
		value: 2,
```

This parameter should not be necessary, as options have all values ​​from 1 to 6 [by default](https://github.com/WordPress/gutenberg/blob/7cf1ceddbef6ea3802014c43ea16d193febcb6d1/packages/block-editor/src/components/block-heading-level-dropdown/index.js#L39).